src/main/resources/assets/js/app/event/UserItemsStopScrollEvent.ts

```diff
@@ -0,0 +1,19 @@
+import {ClassHelper} from 'lib-admin-ui/ClassHelper';
+import {Event} from 'lib-admin-ui/event/Event';
+
+export class UserItemsStopScrollEvent
+    extends Event {
+
+
+    constructor() {
```

Comment:
No need for constructor if the one thing it does is call `super()` with the same arguments.